### ISSUE TITLE
Update S3249: Rule should not raise when the overrides only calls base and has attributes

### ISSUE THREAD

**Issue by Evangelink** [SonarSource/sonar-dotnet#2239]
### Description

The rule should not raise an issue when `GetHashCode` is overridden but only contains a call to object `GetHashCode` AND is annotated with an attribute.

### Repro steps

```csharp
[Browsable(false)]
[EditorBrowsable(EditorBrowsableState.Never)]
public override int GetHashCode() => base.GetHashCode();
```

### Related information

* SonarC# 7.10

We should discuss whether we exclude:

1. Only the case where both `Browsable` and `EditorBrowsable` attributes are present
2. At least `EditorBrowsable`
3. Any attribute

**Comment by Evangelink** [comment]
See https://github.com/SonarSource/sonar-dotnet/issues/1792

cc @sm-g

**Comment by Corniel** [comment]
I would argue any attribute. The only exception you could make (but I wouldn't) that if it only redefines attributes from its base that are defined as being inheritable,
